docroot/sites/all/modules/custom/ppgetstat/ppgetstat.module

```diff
@@ -542,7 +574,7 @@ function ppgetstat_node_insert($node) {
   if (!isset($node->field_user_id[LANGUAGE_NONE][0]['value'])) {
     return;
   }
-  $queue = DrupalQueue::get('stats_jobs');
+  /*$queue = DrupalQueue::get('stats_jobs');
```

Comment:
We need to create views bulk operation then for ability to add users to scanning queue